sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -81,8 +82,77 @@ def get_running_loop():
             loop = asyncio.get_event_loop()
         return loop
 
+class AsyncTransportMixin():
+    def __init__(self):
```

Comment:
A mixin should not have a constructor - and these attributes are only used by websockets anyway - so let's move it to that child constructor

Reply:
done